bridges/TwitterBridge.php

```diff
@@ -546,8 +546,16 @@ private function getApiKey() {
 		if($guestTokenUses === null || !is_array($guestTokenUses) || count($guestTokenUses) != 2
 		|| $guestTokenUses[0] <= 0 || (time() - $refresh) > self::GUEST_TOKEN_EXPIRY) {
 			$guestToken = $this->getGuestToken();
-			$gt_cache->saveData(array(self::GUEST_TOKEN_USES, $guestToken));
-			$r_cache->saveData(time());
+			if ($guestToken === null) {
+				if($guestTokenUses === null) {
+					returnServerError('Could not parse guest token');
```

Comment:
It will raise exception on $this->getGuestToken(), so it won't even return any value.

Reply:
my false i missed a required change.